setup.py

```diff
@@ -71,6 +71,7 @@ def read(*rnames):
         'defusedxml>=0.4.1',
         'Django>=1.8',
         'pysaml2>=4.6.0',
+        'six>=1.13.0',
```

Comment:
This is the version of `six` we are currently pinned to in vndly.  https://github.com/vndly-oss/vndly/blob/master/requirements/base.txt#L139